config-cmd/signin.go

```diff
@@ -0,0 +1,127 @@
+package configCmd
```

Comment:
I'll move it. Wasn't sure it was useful to create a whole package for a single file/command.